src/components/HelloWorld/HelloWorld.test.ts

```diff
@@ -0,0 +1,33 @@
+import { render, cleanup, screen } from '@testing-library/react';
+
+import '@testing-library/jest-dom';
+
+import { HelloWorld, sumFunction } from './HelloWorld';
+import { getByText } from '@testing-library/dom';
+
+afterEach(cleanup);
+
+/**
+ * uncomment code to cover component hello world
+ */
+let container: HTMLElement;
+describe('HelloWorld component suites tests', () => {
+    beforeEach(() => {
+        // const component = render(
+        //     <HelloWorld
+        //         name="Jean"
+        //         title="Hello world"
+        //         description="Lorem ipsum"
+        //     />
+        // );
+        // container = component.container;
```

Comment:
Uncomment this and the test

Reply:
important to add tsx because component use React TSX @see https://stackoverflow.com/questions/62059408/reactjs-and-typescript-refers-to-a-value-but-is-being-used-as-a-type-here-ts